components/scream/tests/scream_shoc/CMakeLists.txt

```diff
@@ -8,5 +8,5 @@ CreateUnitTest(shoc_stand_alone "shoc_stand_alone.cpp" "${NEED_LIBS}" LABELS "sh
 # Copy yaml input file to run directory
 CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/input.yaml
                ${CMAKE_CURRENT_BINARY_DIR}/input.yaml COPYONLY)
-CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/shoc_standalone_ne4_ne4.nc
-               ${CMAKE_CURRENT_BINARY_DIR}/shoc_standalone_ne4_ne4.nc COPYONLY)
+CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/shoc_standalone_ne4_np4.nc
```

Comment:
While we're at it, I think the e3sm convention is to _not_ use an underscore between neX and phys grid specs. That is, `blah_ne4np4` rather than `blah_ne4_np4`.

Reply:
Ahhh yes, good point.  I think we should make the change now and start the good habit.  I'll make the fix and push the changes so we can merge this PR.